uiowa/uiowa
# Repair grad_person_extended split

- Genetics needed person_department, the feature split did not.

## To Test

- sync amcs.uiowa.edu, gss.grad.uiowa.edu and genetics.grad.uiowa.edu and make sure all is dandy config import wise.
